src/lang/en-US.ts

```diff
@@ -18,6 +18,20 @@ const enUSMessages: ResolvedIntlConfig['messages'] = {
     'cta.download': `Download`,
     'cta.login': `Login`,
     'cta.clickHere': `click here`,
+    'cta.moreDetails': `More Details`,
+
+    // Terms
+    'terms.connectors': `Connectors`,
```

Comment:
No clue what to call this... maybe this should just go up with `common`? Basically a place for us to store all the common terms we'll be using.